src/lecturer/components/studentListView/StudentListView.tsx

```diff
@@ -0,0 +1,77 @@
+/* Code adopted from: https://material-ui.com/components/tables/ */
+
+import { makeStyles, Paper, Table, TableBody, TableCell, TableContainer, TableRow } from "@material-ui/core";
+import React, { useEffect, useState } from "react";
```

Comment:
import React will be redundant if you replace in code _React.useState_ with simple _useState_